guide/build-config.md

```diff
@@ -138,13 +139,17 @@ builds:
   scmUrl: https://<external git server>/<repo>.git
 ```
 
-#### environmentId v/s systemImageId
+#### environmentId v/s environmentName v/s systemImageId
 
-The build environment used for builds has a unique id and system image id.
+The build environment used for builds has a unique id, name and system image id.
 
-We allow the user to specify the build environment using either `environmentId` or the `systemImageId`. `environmentId` refers to the exact id of the build environment as stored in the PNC database. This is what we traditionally use, but it is problematic since the ids are different between PNC servers (Devel, Stage, Prod), which makes the `build-config.yaml` not very portable between PNC servers.
+We allow the user to specify the build environment using either `environmentId`, `environmentName` or the `systemImageId`. `environmentId` refers to the exact id of the build environment as stored in the PNC database. This is what we traditionally use, but it is problematic since the ids are different between PNC servers (Devel, Stage, Prod), which makes the `build-config.yaml` not very portable between PNC servers.
 
-The `systemImageId` in contrast, is the same between different PNC servers and should be used if portability is a concern. To get the `environmentId` or the `systemImageId` of the build image, simply run:
+The `environmentName` in contrast, is the same between different PNC servers and should be used if portability is a concern. Environments specified by `environmentName` are always using **the latest updates**, therefore, this is the **preferred** way to specify environments. 
```

Comment:
"preferred and recommended" ?

Reply:
I'd still go with preferred as it is more impactful.

Reply:
ok makes sense, closing